stock_inventory_revaluation/tests/test_stock_inventory_revaluation.py

```diff
@@ -234,6 +234,13 @@ def test_inventory_revaluation_price_change_real(self):
                                  'Incorrect inventory revaluation for '
                                  'type Price Change.')
 
+        quants = self.env['stock.quant'].search([('product_id', '=',
+                                                 self.product_real_1.id)])
+
+        # We should be able to delete the quants afterwards
+        for quant in quants:
+            quant.unlink()
```

Comment:
@jbeficent @gmeficent It's not allowed by odoo to call unlink on a quant... https://github.com/odoo/odoo/blob/9.0/addons/stock/stock.py#L746
UNLESS you call unlink with `force_unlink=True` into the context 
```python
quants..with_context(force_unlink=True).unlink
```
You don't need to loop on the quants to call unlink...